.npmrc

```diff
@@ -0,0 +1,2 @@
+ignore-workspace-cycles=true
+enable-pre-post-scripts=true
```

Comment:
This is optional, but I've opted to keep automatic `pre` and `post` script behaviour so that the repo doesn't change too much in the migration. We can revisit this later.